test/testcases/block/06_codeblock/with_lang_in_fenced_block_name_with_dash.options

```diff
@@ -0,0 +1,2 @@
+:enable_coderay: false
+
```

Comment:
I added this because the HTML generated without the coderay converter is much simpler and is the same like for the existing tests.